### PR TITLE
fix(editor): Remove "move" action from workflow and credential on community plan

### PR DESCRIPTION
**Pull request by cstuncsik**
## Summary

Users on Community edition should not be able to move resources

## Related Linear tickets, Github issues, and Community forum posts

[PAY-1728](https://linear.app/n8n/issue/PAY-1728/bug-move-option-showing-on-instances-with-community-license-without#comment-075026c3)

## Review / Merge checklist

- [x] PR title and summary are descriptive.
- [x] Tests included.

### CODE REVIEW

File: packages/editor-ui/src/components/CredentialCard.test.ts [hunk 1]
```diff
@@ -55,4 +61,31 @@ describe('CredentialCard', () => {
...
+		const controllingId = cardActionsOpener.getAttribute('aria-controls');
+
+		await userEvent.click(cardActions);
+		const actions = document.querySelector(`#${controllingId}`);
+		if (!actions) {
+			throw new Error('Actions menu not found');
+		}
+		await waitFor(() => {
+			expect(actions).toBeInTheDocument();
+		});
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/CredentialCard.test.ts:88]: Wouldn't these two actually check the same thing?  

Maybe you could use
```
await waitFor(() => {
  expect(document.querySelector(`#${controllingId}`)).toBeInTheDocument();
});
```

File: packages/editor-ui/src/components/CredentialCard.vue [hunk 2]
```diff
@@ -62,7 +64,10 @@ const actions = computed(() => {
 		});
 	}
 
-	if (credentialPermissions.value.move) {
+	if (
+		credentialPermissions.value.move &&
+		String(settingsStore.planName).toLowerCase() !== 'community'
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/CredentialCard.vue:69]: Would be worth adding `isCommunityPlan` in the settings. We're likely to reuse this.

File: packages/editor-ui/src/components/WorkflowCard.test.ts [hunk 3]
```diff
@@ -95,10 +98,14 @@ describe('WorkflowCard', () => {
 		});
 
 		const actions = document.querySelector(`#${controllingId}`);
+		if (!actions) {
+			throw new Error('Actions menu not found');
+		}
 		await waitFor(() => {
 			expect(actions).toBeInTheDocument();
 		});
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/WorkflowCard.test.ts:106]: Same question as above.

File: packages/editor-ui/src/components/WorkflowCard.test.ts [hunk 4]
```diff
@@ -138,4 +145,31 @@ describe('WorkflowCard', () => {
...
+		const controllingId = cardActionsOpener.getAttribute('aria-controls');
+
+		await userEvent.click(cardActions);
+		const actions = document.querySelector(`#${controllingId}`);
+		if (!actions) {
+			throw new Error('Actions menu not found');
+		}
+		await waitFor(() => {
+			expect(actions).toBeInTheDocument();
+		});
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/WorkflowCard.test.ts:172]: Same question as above

File: packages/editor-ui/src/components/WorkflowCard.vue [hunk 5]
```diff
@@ -93,7 +93,10 @@ const actions = computed(() => {
 		});
 	}
 
-	if (workflowPermissions.value.move) {
+	if (
+		workflowPermissions.value.move &&
+		String(settingsStore.planName).toLowerCase() !== 'community'
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/WorkflowCard.vue:98]: Same as previous (reusability).

**Review by alexgrozav — CHANGES_REQUESTED** [review]
Looks great overall! Have a few small remarks.

File: packages/editor-ui/src/components/CredentialCard.test.ts (hunk 1 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/CredentialCard.test.ts]: It wasn't necessary, I have removed

File: packages/editor-ui/src/components/CredentialCard.vue (hunk 2 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/CredentialCard.vue]: Added the new getter to settings store

**Review by alexgrozav — DISMISSED** [review]

**Review by alexgrozav — DISMISSED** [review]

**Review by alexgrozav — APPROVED** [review]

**Pull request merged**